assets/src/application/ui/input/Button/style.scss

```diff
@@ -0,0 +1,494 @@
+
+:root {
+	--ee-icon-button-size: 36px;
+}
+
+.ee-event-editor {
+	.ee-button.ee-button {
+		border-radius: var(--ee-border-radius-small);
+		border-style: solid;
+		border-width: 1px;
+		border-bottom: none;
+		box-shadow: var(--ee-box-shadow-tiny);
+		color: var(--ee-text-on-grey-14) !important;
+		cursor: pointer;
+		display: inline-flex;
+		font-family: 'Segoe UI', Candara, 'Bitstream Vera Sans', 'DejaVu Sans', 'Bitstream Vera Sans', 'Trebuchet MS',
+			Verdana, 'Verdana Ref', sans-serif;
+		font-size: var(--ee-font-size-default);
+		font-weight: bold;
+		line-height: var(--ee-font-size-default);
+		margin: 0 0 var(--ee-margin-smaller);
+		outline: none !important;
+		position: relative;
+		text-decoration: none;
+		text-align: center;
+		text-shadow: var(--ee-text-shadow-inset-light);
+		top: 0;
+		transition: border-color 0.1s ease 0.1s, color 0.1s ease 0.1s !important;
+		vertical-align: middle;
+
+		&:active,
+		&:not([disabled]):active,
+		&:focus,
+		&:visited {
+			border-radius: var(--ee-border-radius-small);
+			border-style: solid;
+			border-width: 1px;
+			box-shadow: var(--ee-box-shadow-tiny);
+			cursor: pointer;
+			display: inline-block;
+			font-family: 'Segoe UI', Candara, 'Bitstream Vera Sans', 'DejaVu Sans', 'Bitstream Vera Sans',
+				'Trebuchet MS', Verdana, 'Verdana Ref', sans-serif;
+			font-size: var(--ee-font-size-default);
+			font-weight: bold;
+			line-height: var(--ee-font-size-default);
+			// margin: 0 0 var(--ee-margin-smaller); Setting margins on these states is a bit confusing, needs discussion.
+			outline: none !important;
+			position: relative;
+			text-decoration: none;
+			text-align: center;
+			text-shadow: var(--ee-text-shadow-inset-light);
+			top: 0;
+			transition: border-color 0.1s ease 0.1s, color 0.1s ease 0.1s !important;
+			vertical-align: middle;
+		}
+
+		&:active {
+			box-shadow: none;
+		}
+
+		&:hover {
+			&:not(:disabled):not([aria-disabled='true']):not(.is-secondary):not(.is-primary):not(.is-tertiary):not(.is-link) {
+				background-color: rgb(255, 255, 255);
+				color: rgb(25, 30, 35);
+				box-shadow: inset 0 0 0 1px #555d66, inset 0 0 0 2px #fff;
+			}
+		}
+
+		&:focus {
+			box-shadow: 0 0 0 1px var(--ee-color-primary-low-contrast), 0 2px 2px rgba(0, 0, 0, 0.6);
+			&:not(:disabled) {
+				background-color: rgb(255, 255, 255);
+				color: rgb(25, 30, 35);
+				box-shadow: inset 0 0 0 1px #6c7781, inset 0 0 0 2px #fff;
+				outline: 2px solid rgba(0, 0, 0, 0);
+			}
+		}
+
+		&:not(.ant-btn-icon-only),
+		&:active:not(.ant-btn-icon-only),
+		&:focus:not(.ant-btn-icon-only),
+		&:visited:not(.ant-btn-icon-only) {
+			// box-sizing: content-box !important; // Need to ask/ investigate this
+			// min-height: var(--ee-font-size-big); // Also this is questionable
+			// padding: var(--ee-padding-tiny) var(--ee-padding-default) var(--ee-padding-tiny) var(--ee-padding-small); // Also this is questionable
```

Comment:
@tn3rb I need your :brain: here, because I am not aware of the previous context;
are these still actual in event editor? because these are causing some trouble